From 4ebeb9d5e485a61d76b87a9db9a9d31ceb07514a Mon Sep 17 00:00:00 2001
From: Anjaneyulu Reddy Mule <anjaneyulu.reddy.mule@xilinx.com>
Date: Wed, 29 Jul 2020 19:48:57 +0530
Subject: [PATCH] updated create qp data structure With Memory manager in
 place, the providers library provides queue base addresses to the ERNIC
 driver

Signed-off-by: Anjaneyulu Reddy Mule <anjaneyulu.reddy.mule@xilinx.com>
---
 include/uapi/rdma/xib-abi.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/uapi/rdma/xib-abi.h b/include/uapi/rdma/xib-abi.h
index 42cb8fe..aeb7c3b 100755
--- a/include/uapi/rdma/xib-abi.h
+++ b/include/uapi/rdma/xib-abi.h
@@ -42,8 +42,10 @@ struct xib_ib_create_cq {
 };
 
 struct xib_ib_create_qp {
-	__aligned_u64 buf_addr;
 	__aligned_u64 db_addr;
+	__aligned_u64 sq_ba;
+	__aligned_u64 rq_ba;
+	__aligned_u64 cq_ba;
 };
 
 struct xib_ib_alloc_pd_resp {
-- 
2.7.4

